semantic/polytype.go

```diff
@@ -16,6 +16,8 @@ type PolyType struct {
 	fb *fbsemantic.PolyType
 }
 
+func (p PolyType) Fb() *fbsemantic.PolyType { return p.fb }
```

Comment:
This a small thing, but the caller of this method does not actually need to use the `*fbsemantic.PolyType` for anything, it just needs to know if the pointer is `nil`.  Can you change the name of this method to `IsNil()` and make it return a `bool`?This way we can continue completely hide the underlying representation.  

Also, since this is an exported method, it would be good to have a comment for it.

Reply:
Ok, I'll change that.